app/components/ObservationsInspector.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import {
+  Card,
+  Classes,
+  Drawer,
+  Elevation,
+  Position,
+  Tooltip
+} from '@blueprintjs/core';
+import { useTranslation } from 'react-i18next';
+import path from 'path';
+import CreatableSelect from 'react-select/creatable';
+
+import { formatAnimalClassName } from '../constants/animalsClasses';
+import { taxonOptions } from '../constants/taxons';
+
+type ObservationCardProps = {
+  observation: Observation;
+  predictionOverride?: CreatableOption;
+  onPredictionOverride: (location: string, prediction: CreatableOption) => void;
+};
+
+function ObservationCard(props: ObservationCardProps) {
+  const { observation, predictionOverride, onPredictionOverride } = props;
+  const { t } = useTranslation();
+
+  const handlePredictionOverride = (newValue: CreatableOption) => {
+    onPredictionOverride(observation.location, newValue);
+  };
+
+  const predictions = [
+    [formatAnimalClassName(observation.pred_1), observation.score_1],
+    [formatAnimalClassName(observation.pred_2), observation.score_2],
+    [formatAnimalClassName(observation.pred_3), observation.score_3]
+  ];
+
+  return (
+    <Card
```

Comment:
I'd suggest splitting this into several variables that you later merge to make the high level structure more readable in the final return

Reply:
Done.